Makefile

```
@@ -8,5 +8,8 @@ peervpn.o: peervpn.c
 
 install:
 	install peervpn /usr/local/sbin/peervpn
+	install peervpn.conf /etc/peervpn.conf
+	install systemd/peervpn.service /etc/systemd/system/peervpn.service
```

As far as I understand systemd packages should install their unit files into /usr/lib/systemd/system/ so that /etc/systemd/system can be used by the system administrator to override the packaged unit files.

```suggestion
	install systemd/peervpn.service /usr/lib/systemd/system/peervpn.service
```

---

As long as it is not packaged for a system, /etc/systemd/system is the right place. You can still override the service file.

Btw your override systemd file can include your `ExecStartPost=/opt/peervpnroutes.sh` but the file in this repository should never have this very specific option for your environment.

I am running this in my environment:
```
[Unit]
Description=PeerVPN network service
Wants=network-online.target
After=network-online.target
ConditionPathExists=/etc/peervpn/peervpn.conf

[Service]
Type=simple
ExecStart=/usr/local/sbin/peervpn /etc/peervpn/peervpn.conf
Restart=always
WorkingDirectory=/etc/peervpn
ProtectSystem=strict

[Install]
WantedBy=multi-user.target
```

Then you are overriding something with `systemctl edit  peervpn.service` which creates the file in `/etc/systemd/system/peervpn.service.d/override.conf` where you can add your 
```
[Service]
ExecStartPost=/opt/peervpnroutes.sh
```